Orange/tests/test__orange.py

```diff
@@ -13,10 +13,6 @@ def test_valuecount(self):
         b = _valuecount.valuecount(a)
         np.testing.assert_almost_equal(b, [[1, 2], [0.6, 0.4]])
 
-        a = np.array([[0, 1, 1, 1], [0.1, 0.2, 0.3, 0.4]])
-        b = _valuecount.valuecount(a)
-        np.testing.assert_almost_equal(b, [[0, 1], [0.1, 0.9]])
```

Comment:
While I see that the same `a` was already used, This assert line was never duplicated and you basically just removed one test if I'm reading this correctly.

The only duplicate lines here that should have been removed are:

```
a = np.array([[0, 1, 1, 1], [0.1, 0.2, 0.3, 0.4]])
b = _valuecount.valuecount(a)
```
But I don't understand why the different assert statements still pass.